.github/workflows/docs.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: make -C doc html
 
       - name: Deploy
-        uses: peaceiris/actions-gh-pages@v4
+        uses: peaceiris/actions-gh-pages@v3
```

Comment:
not sure if we want to downgrade this.  Let's discuss.  In genera its better to go with a higher version unless there is a reason to downgrade.  Please ask to the whole group on slack. If the new version works we can update the cookiecutter.